CHANGELOG.md

```diff
@@ -2,9 +2,11 @@
 
 ## Master (Unreleased)
 
+- Fix an error when using `RSpec/FilePath` and revert to enabled by default. If you have already migrated `RSpec/SpecFilePathSuffix` and `RSpec/SpecFilePathFormat`, no problem. The `RSpec/FilePath` before migration and the `RSpec/SpecFilePathSuffix` and `RSpec/SpecFilePathFormat` at the destination are available respectively. ([@ydah])
+
 ## 2.24.0 (2023-09-08)
 
-- Split `RSpec/FilePath` into `RSpec/SpecFilePathSuffix` and `RSpec/SpecFilePathFormat`. `RSpec/FilePath` cop is enabled by default, the two new cops are pending and need to be enabled explicitly. ([@ydah])
+- **(Breaking)** Split `RSpec/FilePath` into `RSpec/SpecFilePathSuffix` and `RSpec/SpecFilePathFormat`. `RSpec/FilePath` cop is disabled by default and the two new cops are pending and need to be enabled explicitly. ([@ydah])
```

Comment:
I am confused. Isn't `RSpec/FilePath` now enabled by default again? :thinking: If so, is this really a breaking change then? :nerd_face: 

```suggestion
- Split `RSpec/FilePath` into `RSpec/SpecFilePathSuffix` and `RSpec/SpecFilePathFormat`. `RSpec/FilePath` cop is enabled by default and the two new cops are pending and need to be enabled explicitly. ([@ydah])
```

Reply:
Oh, sure, we'll enable it back, but isn't it is breaking change for those who use v2.24.0?

Reply:
Perhaps we’re asking “does two breaking changes equal zero breaking changes”?

Reply:
Ah, I understand. Indeed it is.

Reply:
I updated this PR. How about it?

Reply:
Feedback: Our builds were actually broken twice by this two breaking changes.